lambda/src/main/java/com/lambda/IssueUpdater.java

```diff
@@ -57,22 +65,50 @@ public Issue setActualHours(final int issueId) {
                     LocalDateTime.now());
         }
         DecimalFormat df = new DecimalFormat("0.0");
-        float actualHours = Float.parseFloat((df.format(elapsed.toMinutes() / 60)));
+        float actualHours = Float.parseFloat((df.format(elapsed.toMinutes() / 60.0)));
 
         if (actualHours > 999 || actualHours < 0)
             return null;
 
         return client.updateIssue(new UpdateIssueParams(issue.getId()).actualHours(actualHours));
     }
 
+    private Duration calculateWorkingHours(LocalDateTime start, LocalDateTime end) {
+        LocalDateTime current = start;
+        Duration totalWorkingDuration = Duration.ZERO;
+
+        while (current.isBefore(end)) {
+            if (isWeekday(current)) {
+                LocalDateTime endOfDay = current.withHour(19).withMinute(30); // End at 17:30 ITC as 19:30 JST
+                
+                // Add hours for current day or until end time
+                LocalDateTime actualEnd = isSameDate(end, endOfDay) ? end : endOfDay;
+                totalWorkingDuration = totalWorkingDuration.plus(Duration.between(current, actualEnd));
+            }
+
+            // Move to the next day at 09:30 JST
+            current = current.plusDays(1).withHour(9).withMinute(30);
```

Comment:
[suggestion]
I prefer setting the start and end time as constants.

```java
private static final LocalTime WORK_START_TIME = LocalTime.of(9, 30);

current = current.plusDays(1).atTime(WORK_START_TIME);
```

Reply:
I change on this [98be012](https://github.com/FaberTechnology/backlog-time-recorder/pull/1/commits/98be012467c31d412bb41beab4058546e14bda7d)